JavaHackerrank/Solving_For_Queries_With_Cups.java

```diff
@@ -0,0 +1,143 @@
+import java.io.*;
```

Comment:
Problem Link : https://www.hackerrank.com/contests/hack-the-interview-vi/challenges/the-cup-game/problem